packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-soc-idp/go/integrationsteps.md

```diff
@@ -0,0 +1,163 @@
+## Integration steps
+
+### Step 1: Create the SDK client on application load
+
+When the user navigates to the home page and the application loads, create a new
+SDK Client object by calling the `NewClient` method.
+
+```go
+idx, err := idx.NewClient(
+      c.Okta.IDX.ClientID,
+      c.Okta.IDX.ClientSecret,
+      c.Okta.IDX.Issuer,
+      c.Okta.IDX.Scopes,
+      c.Okta.IDX.RedirectURI)
+if err != nil {
+    log.Fatalf("new client error: %+v", err)
+}
+```
+
+### Step 2: Build sign-in page and initialize client object
```

Comment:
"Build the sign-in page and initialize the client object"

Reply:
Updated to "Step 2: Navigate to the sign-in page"